docs/database/elasticsearch.md

```diff
@@ -14,12 +14,7 @@ described on this page.
 
 ## Span Name
 
-The **span name** SHOULD be of the format `<endpoint id>`.
-
-The elasticsearch endpoint identifier is used instead of the url path in order to reduce the cardinality of the span
-name, as the path could contain dynamic values. The endpoint id is the `name` field in the
-[elasticsearch schema](https://raw.githubusercontent.com/elastic/elasticsearch-specification/main/output/schema/schema.json).
-If the endpoint id is not available, the span name SHOULD be the `http.request.method`.
+The **span name** follows the [general database span name guidelines](database-spans.md#name) with the endpoint identifier stored in `db.operation.name`, and the index stored in `db.collection.name`.
```

Comment:
is this needed? it seems like it is already following the general database span name guidelines? (and documenting what to put into `db.operation.name` and `db.collection.name` for elasticsearch can be done in overrides for those attributes)

Reply:
This info is definitely covered in the attributes, so in that sense we'd not loose any information if we drop this line. My thinking was that calling out specifically that we put the endpoint id into the span name could be useful for readers.

If we drop this part, then there is nothing said in `elasticsearch.md` about the span name. And since this is a bit different from other databases, I thought it's worth calling out. Otherwise the reader needs to put together the fact, that operation name is something special (the endpoint id) for Elasticsearch and that is the one in the span name.

I just though it's worth calling out that we expect span names in `endpoint id + index` format at the top of the `.md` file.

Reply:
> And since this is a bit different from other databases, I thought it's worth calling out. Otherwise the reader needs to put together the fact, that operation name is something special (the endpoint id) for Elasticsearch and that is the one in the span name.

In the context of the span name there is no difference though? The difference is captured in the corresponding attribute briefs/notes.

Reply:
Yes, that's true.

I thought this is worth calling out extra. For people familiar with SemConv this is obvious, but if someone just implements this spec, I think that short sentence is very useful.

But if others think this is not needed I'm ok dropping this.

Reply:
I'm ok either way, we can always change our minds later as we build out more database system specific conventions since this kind of change can be done post stability

Reply:
I believe this discussion is resolved.